compiler/rustc_target/src/spec/apple_base.rs

```diff
@@ -114,3 +114,8 @@ pub fn ios_sim_llvm_target(arch: &str) -> String {
     let (major, minor) = ios_deployment_target();
     format!("{}-apple-ios{}.{}.0-simulator", arch, major, minor)
 }
+
+pub fn macabi_llvm_target(arch: &str) -> String {
+    let (major, minor) = ios_deployment_target();
+    format!("{}-apple-ios{}.{}.0-macabi", arch, major, minor)
```

Comment:
Not sure whether this is suitable for the catalyst use case...

Please feel free to give hints.